pkg/storaged/stratis-details.jsx

```diff
@@ -207,6 +217,140 @@ export const StratisPoolDetails = ({ client, pool }) => {
         });
     }
 
+    function add_passphrase() {
+        dialog_open({
+            Title: _("Add passphrase"),
+            Fields: [
+                PassInput("passphrase", _("Passphrase"),
+                          { validate: val => !val.length && _("Passphrase cannot be empty") }),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230706-092324-4f3ab2f3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)